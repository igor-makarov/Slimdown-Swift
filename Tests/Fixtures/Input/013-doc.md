# Title

And *now* [a link](http://www.google.com) to **follow** and [another](http://yahoo.com/).

* One
* Two
* Three

## Subhead

One **two** three **four** five.

1. One
2. Two
3. Three

More text with `inline($code)` sample.

> A block quote
> across two lines.

More text...