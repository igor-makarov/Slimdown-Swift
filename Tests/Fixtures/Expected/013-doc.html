<h1>Title</h1>

<p>And <em>now</em> <a href='http://www.google.com'>a link</a> to <strong>follow</strong> and <a href='http://yahoo.com/'>another</a>.</p>

<ul>
	<li>One</li>

	<li>Two</li>

	<li>Three</li>
</ul>
<h2>Subhead</h2>

<p>One <strong>two</strong> three <strong>four</strong> five.</p>

<ol>
	<li>One</li>

	<li>Two</li>

	<li>Three</li>
</ol>

<p>More text with <code>inline($code)</code> sample.</p>

<blockquote>A block quote</blockquote>
<blockquote>across two lines.</blockquote>

<p>More text...</p>